src/test/unit/project-converter_test.ts

```diff
@@ -910,10 +910,14 @@ export function increment() {
       });
       assertSources(await convert(), {
         'test.js': `
+/**
```

Comment:
if comment is empty like this, we should continue to remove it

Reply:
Seems reasonable.